doc/ddu.txt

```diff
@@ -1607,7 +1630,13 @@ CREATE KIND                                                  *ddu-create-kind*
 
 To create kind, you should read other kinds implementation.
 
-The kinds must put under "denops/@ddu-kinds/*.ts".
+The kinds must be placed in one of the following structures, where {name} is
+the kind name:
+
+	denops/@ddu-kinds/{name}.ts
+	denops/@ddu-kinds/{name}/main.ts
+
+NOTE: If both structures exist, "{name}.ts" is preferred.
```

Comment:
There is no description in the documentation that the "deno.json[c]" and "import_map.json[c]" files are also loaded.